app/views/catalog/_home_page.html.erb

```diff
@@ -58,26 +58,26 @@
         </header>
         <ul class="list-unstyled">
           <li class="mb-3">
-            <div class="fw-bold">Start with a simple keyword search</div>
-            Examples: “the new york times” or “stephan graham thesis”
+            <span class="fw-medium">Start with a simple keyword search</span>
```

Comment:
Why did these change from div to span?  Don't we want a block element here?  Doesn't the `<p>` added below add extra margin at the bottom?